Makefile-daemon.am

```diff
@@ -1,11 +1,12 @@
 dbus_built_sources = rpm-ostreed-generated.h rpm-ostreed-generated.c
 
 rpm-ostreed-generated.h: rpm-ostreed-generated.c
-rpm-ostreed-generated.c: Makefile.am $(top_srcdir)/src/daemon/org.projectatomic.rpmostree1.xml
+rpm-ostreed-generated.c: Makefile $(top_srcdir)/src/daemon/org.projectatomic.rpmostree1.xml
 	$(AM_V_GEN) gdbus-codegen \
 		--interface-prefix org.projectatomic.rpmostree1 \
 		--c-namespace RPMOSTree \
 		--c-generate-object-manager \
+	  --c-generate-autocleanup=all \
```

Comment:
Ahh no wonder I couldn't find this on my machine. This was added in Glib 2.50, but F24 is on 2.48. I will upgrade soon (or at least switch to containers), though F24 *is* still supported, right? I guess this depends on the outcome of https://pagure.io/atomic-wg/issue/228, though I'd like to be able to keep releasing to F24 if we're gonna keep running composes for it. Can we leave this off for now?